terraform/deployments/concourse-iam/main.tf

```diff
@@ -27,7 +27,7 @@ provider "aws" {
 }
 
 resource "aws_iam_role" "govuk_concourse_deployer" {
-  name        = "govuk-concourse-deployer"
+  name        = "govuk-concourse-deployer-${terraform.workspace}"
```

Comment:
👍 we'll also need to change `govuk-concourse-deployer` in the Concourse pipelines to point at the default workspace.

Reply:
Or we can use an interpolation to avoid changing the name of the role for default?

Reply:
as Richard pointed out, this probably doesn't need to be workspace aware, reverted to its original value